packages/p2p-v2/src/components/ClickableText/ClickableText.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ComponentProps, PropsWithChildren } from 'react';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import './ClickableText.scss';
+
+type TClickableTextProps = ComponentProps<typeof Text> & {
+    onClick?: () => void;
+};
+
+const ClickableText = ({ children, onClick, ...props }: PropsWithChildren<TClickableTextProps>) => {
```

Comment:
how about renaming this component to `Link`?

Reply:
Won't that be confusing @farrah-deriv? because if a component was named `Link`, I'd expect it to have an `href` or a `to` `attribute` (`prop`), but this doesn't. If you still think I should rename it to `Link`, the I will. Please let me know what you decide.

Reply:
noted @jim-deriv, i just thought it has the same behavior as a link